src/plugins/enzymes/src/FindEnzymesAlgorithm.h

```diff
@@ -90,7 +89,7 @@ class FindEnzymesAlgorithm {
                 buf.append(dnaseq.mid(startPos));
                 buf.append(dnaseq.mid(0, size));
                 for (int s = 0; s < size; s++) {
-                    bool match = matchSite(buf.constData() + s, pattern, plen, unknownChar, fn);
+                    bool match = matchSite(buf.constData() + s, pattern, plen, fn);
```

Comment:
The change makes the algorithm to produce wrong results. Please add more tests.
Before:
![image](https://github.com/ugeneunipro/ugene/assets/17791039/1d4d047a-2bd0-46df-ab9b-c25bd68b3140)

After:
![image](https://github.com/ugeneunipro/ugene/assets/17791039/069f087b-46ff-4470-b0ca-139c800558cd)

Reply:
Fixed.

Reply:
test010.xml